compat.c

```diff
@@ -33,3 +33,18 @@ CreateTemplateTupleDescCompat(int nattrs, bool hasoid)
 	return CreateTemplateTupleDesc(nattrs, hasoid);
 #endif
 }
+
+inline void InitPostgresCompat(const char *in_dbname, Oid dboid,
```

Comment:
Please, follow here BSD style of code formatting - function type has to be located on preceded line from its name definition.

Reply:
done